external/downstream/pasture/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
-if(ENABLE_psi4pasture)
-  ExternalProject_Add(psi4pasture
-    DEPENDS psi4_core
+if(ENABLE_pasture)
+  ExternalProject_Add(pasture
+    DEPENDS psi4-core
     GIT_REPOSITORY https://github.com/psi4/psi4pasture
     CMAKE_ARGS -DENABLE_ccsort=${ENABLE_ccsort}
                -DENABLE_transqt2=${ENABLE_transqt2}
```

Comment:
You're absolutely right, @amjames. I was thinking of "my way" for plugins (`>>> cmake -C {}/psi4PluginCache.cmake -DCMAKE_PREFIX_PATH={} .`) where the prefix path catches everything. I've made the correction. Just patching up the docs a bit before pushing.